### DISCUSSION TITLE
Benchmark for standard MRI reconstruction

### DISCUSSION THREAD

**Discussion by aTrotier** [JuliaMath/NFFT.jl#71] [Ideas]
Hi,
Thanks to @tknopp we know have access to a large number of non uniform FFT implementation.

What are your thoughts about some MRI specific benchmarks to help the user choose the right one for example :
- 2D and 3D acquisition
- Large matrix > 256^3
- Multi-channel reconstruction (don't know if it will make a difference)
- CPU / GPU

Aurélien

**Comment by tknopp** [comment]
I am already working on it: https://github.com/JuliaMath/NFFT.jl/tree/master/benchmark

It will not be MRI specific though but I will focus on common sizes used in the NFFT literature. 

At this point what is more important is the interface and the correctness. So I use these wrappers to test if `AbstractNFFTs` has the right interface.
